jersey-servers/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     compile "com.fasterxml.jackson.module:jackson-module-afterburner"
     compile "com.jcraft:jzlib"
     compile "com.palantir.safe-logging:safe-logging"
+    compile "com.palantir.tokens:auth-tokens"
```

Comment:
also don't we want auth-tokens2?

one option is to add the param converter to auth-tokens...